app/src/main/java/com/example/bookish/composables/screens/Settings.kt

```diff
@@ -1,31 +1,121 @@
-package com.example.bookish.composables.screens
-
-import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.material.Text
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.unit.sp
-import com.example.bookish.R
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 
 @Composable
 fun SettingsScreen() {
-    Box(
-        modifier = Modifier
-            .fillMaxSize()
-            .background(Color.Black),
-        contentAlignment = Alignment.Center
+    Surface(modifier = Modifier.fillMaxSize()) {
+        Column(modifier = Modifier.padding(16.dp)) {
+            Text(
+                text = "Settings",
+                style = MaterialTheme.typography.h4,
+                modifier = Modifier.padding(bottom = 16.dp)
+            )
+            Text(
+                text = "Display Settings",
+                style = MaterialTheme.typography.subtitle1,
+                modifier = Modifier.padding(bottom = 8.dp)
+            )
+            DropDownSetting("Font Size", listOf("Small", "Medium", "Large"))
+            SwitchSetting("Theme", true)
+            DropDownSetting("Margin Size", listOf("Small", "Medium", "Large"))
+
+            Spacer(modifier = Modifier.height(32.dp))
+
+            Text(
+                text = "Reading Preferences",
+                style = MaterialTheme.typography.subtitle1,
+                modifier = Modifier.padding(bottom = 8.dp)
+            )
+            SwitchSetting("Auto Night Mode", true)
+            DropDownSetting("Highlight Color", listOf("Yellow", "Green", "Blue"))
+            DropDownSetting("Reading Progress", listOf("Page Number", "Percentage"))
+
+            Spacer(modifier = Modifier.weight(1f))
+
+            Button(
+                onClick = { /* Save settings logic */ },
```

Comment:
As this is mostly a design pool request, you can not implement these actions just yet. But please, do not forget about settings functionality!

Reply:
Sure, thanks for the reminder!